spec/rails_helper.rb

```diff
@@ -83,14 +83,19 @@
 
   config.before :each do |example|
     unless example.metadata[:elasticsearch]
-      es = Class.new do
+      ::FakeEsClient ||= Class.new do
+        def exists(*args); false; end
+      end
+
+      ::FakeEsProxy ||= Class.new do
         def index_document; end
         def update_document; end
         def delete_document; end
-      end.new
+        def client; FakeEsClient.new; end
+      end
 
       searchable_models.each do |model|
-        allow_any_instance_of(model).to receive(:__elasticsearch__).and_return(es)
+        allow_any_instance_of(model).to receive(:__elasticsearch__).and_return(FakeEsProxy.new)
```

Comment:
I simply stub another ES client method. Somehow I triggered it in a newly added spec. This is not relevant to specs which actually hit ES.